### DISCUSSION TITLE
Variance Arrows — Display / Interface / Panel

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#5964] [Feature Requests]
The idea is to add a Display / Interface / Panel that can show variance (KPI arrows) as trend arrows and a change percentage. I'm thinking this can be done with the following options:

* `value` — to compare
* `datetime` — for period comparison
* `period` — eg: year, month, week, day, etc
* `show_arrow` — toggles arrow in output
* `show_delta` — toggles delta value in output
* `show_percent` — toggles delta percent in output

The component would have to get the current `value`, but also compare that against the `value` at the end of the previous period, and then calculate the trend and delta (value or percent). Not sure how performant this would be, or if we can improve it by making it an actual field (instead of an alias).

![Screen Shot 2021-05-31 at 5 44 41 PM](https://user-images.githubusercontent.com/522079/120244314-e8396f00-c237-11eb-9c9f-926ceb7b77f0.png)

[Maaaybe we can even leverage SQL (the API) for this.](https://stackoverflow.com/questions/14717393/sql-query-calculating-the-deltas-in-a-time-series)

**Comment by azrikahar** [comment]
#9773 somewhat covers the trend arrow & color aspect of this feature request via Conditional Styles, but understandably not including the delta/percentage & comparisons.

**Comment by azrikahar** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
